cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -54,11 +54,17 @@ service PackagesService {
   //     put: "/core/packages/v1alpha1/installedpackages"
   //   };
   // }
-  // rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
-  //   option (google.api.http) = {
-  //     delete: "/core/packages/v1alpha1/installedpackages"
-  //   };
-  // }
+  rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
+    option (google.api.http) = {
+      // Example:
+      // /core/packages/v1alpha1/installedpackages/plugin/helm.packages/v1alpha1/ns/my-ns/release-identifier
+      delete: "/core/packages/v1alpha1/installedpackages/plugin/{installed_package_ref.plugin.name}/{installed_package_ref.plugin.version}/ns/{installed_package_ref.context.namespace}/{installed_package_ref.identifier}"
+      // We allow a body for the delete request so that we can support multi-cluster
+      // environments, where the context.cluster is also set, without requiring the
+      // cluster in the URL.
```

Comment:
Given that Kubeapps may not be wanted as a multi-cluster packaging solution, I'm a little hesitant to include the cluster in the URL. Instead, it'll be optional as it is already for the grpc calls, but can be specified in the http body.